hotsos/core/plugins/openstack/__init__.py

```diff
@@ -878,6 +879,60 @@ def release_name(self):
 
         return relname
 
+    @property
+    def apache2_ssl_config_file(self):
+        return os.path.join(HotSOSConfig.DATA_ROOT,
+                            'etc/apache2/sites-enabled',
+                            'openstack_https_frontend.conf')
+
+    @property
+    def ssl_enabled(self):
```

Comment:
you could shorted this to:

return os.path.exists(self.apache2_ssl_config_file)

Reply:
fixed on 59fc4895706a98108a6fc7c588544aceb54b1787